pkg/virt-api/rest/subresource.go

```diff
@@ -401,6 +410,35 @@ func (app *SubresourceAPIApp) RestartVMRequestHandler(request *restful.Request,
 		return
 	}
 
+	if forceRestart == "True" {
```

Comment:
this might fail, it's safe to use `ParseBool`, please change that.
https://golang.org/pkg/strconv/#hdr-Numeric_Conversions

Reply:
corrected

Reply:
did you push that? :P

Reply:
Just pushed ;)